cmd/helpMessages.go

```diff
@@ -3,283 +3,435 @@ package cmd
 import "github.com/Azure/azure-storage-azcopy/common"
 
 // ===================================== ROOT COMMAND ===================================== //
-const rootCmdShortDescription = "AzCopy is a command line tool that moves data into/out of Azure Storage."
+const rootCmdShortDescription = "AzCopy is a command-line tool that moves data into and out of Azure Storage."
 
 const rootCmdLongDescription = "AzCopy " + common.AzcopyVersion +
 	`
-Project URL: github.com/Azure/azure-storage-azcopy
+  The general format of the commands is: ``azcopy [command] [arguments] --[flag-name]=[flag-value]``.
```

Comment:
>`` [](start = 41, length = 2)

this file doesn't compile any more.

Please try installing Go here ([https://golang.org/](https://golang.org/)) and compile the tool using: go build -o azcopy.exe, and make sure the help messages are valid Go strings

Reply:
Ok. I fixed all errors and compiled successfully. I had to add some new constants to support the formatting of code blocks in the output.